### PR TITLE
Fix namespacing

### PR DESCRIPTION
**Pull request by YingboMa**
Requires https://github.com/SciML/ModelingToolkit.jl/pull/2249

### CODE REVIEW

**Review by isaacsas — APPROVED** [review]
Thanks! Feel free to bump this once the MTK PR gets merged.

File: Project.toml [hunk 1]
```diff
@@ -30,7 +30,7 @@ JumpProcesses = "9.3.2"
 LaTeXStrings = "1.3.0"
 Latexify = "0.14, 0.15, 0.16"
 MacroTools = "0.5.5"
-ModelingToolkit = "8.47"
+ModelingToolkit = "8.65"
```
Comment (isaacsas) [inline Project.toml:33]: Should this be 8.66?

Comment (YingboMa) [reply Project.toml:33]: Ah, you are right.

**Comment by isaacsas** [comment]
Thanks! I'll merge and release a new version once the doc build finishes ok.

**Comment by YingboMa** [comment]
Great! Thanks!

**Pull request merged**